ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -3,17 +3,52 @@
 
 package com.daml.platform.store.backend
 
+import java.io.InputStream
 import java.sql.Connection
+import java.time.Instant
 
-import com.daml.ledger.participant.state.v1.Offset
+import com.daml.ledger.{ApplicationId, TransactionId}
+import com.daml.ledger.api.domain.{LedgerId, ParticipantId, PartyDetails}
+import com.daml.ledger.api.v1.command_completion_service.CompletionStreamResponse
+import com.daml.ledger.participant.state.index.v2.PackageDetails
+import com.daml.ledger.participant.state.v1.{Configuration, Offset}
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.PackageId
+import com.daml.lf.ledger.EventId
+import com.daml.platform
 import com.daml.platform.store.DbType
+import com.daml.platform.store.appendonlydao.events.{ContractId, EventsTable, Key, Raw}
+import com.daml.platform.store.backend.StorageBackend.RawTransactionEvent
 import com.daml.platform.store.backend.postgresql.PostgresStorageBackend
+import com.daml.platform.store.entries.{ConfigurationEntry, PackageLedgerEntry, PartyLedgerEntry}
+import com.daml.platform.store.interfaces.LedgerDaoContractsReader.KeyState
+import com.daml.scalautil.NeverEqualsOverride
+
+import scala.util.Try
 
 /** Encapsulates the interface which hides database technology specific implementations for parallel ingestion.
```

Comment:
Can you update this brief? It just mentions parallel ingestion, whereas the adapted interface is much wider now.

Reply:
sure